modules/test/playwright/tests/osb-faro-web/utils/navigation.ts

```diff
@@ -8,6 +8,15 @@ import {Page} from '@playwright/test';
 import {faroConfig} from '../faro.config';
 import {waitForLoading} from './loading';
 
+export enum ACPage {
+	sitePage = 'sites',
+	assetPage = 'assets',
+	eventAnalysisPage = 'events',
+	segmentPage = 'segments',
+	individualPage = 'individuals',
+	testPage = 'tests',
+}
+
```

Comment:
Hey @marcosapmf, I think I put the URL term for the individuals and segments pages wrong in the other comment. Here is the correct one

```suggestion
export enum ACPage {
	sitePage = 'sites',
	assetPage = 'assets',
	eventAnalysisPage = 'event-analysis',
	segmentPage = 'contacts/segments',
	individualPage = 'contacts/individuals',
	testPage = 'tests',
}

```